pkg/memlogd/cmd/logread/main.go

```diff
@@ -49,7 +61,13 @@ func main() {
 		panic(err)
 	}
 
-	r := bufio.NewReader(conn)
-	r.WriteTo(os.Stdout)
+	var entry LogEntry
+	decoder := json.NewDecoder(conn)
+	for {
+		if err := decoder.Decode(&entry); err != nil {
+			panic(err)
```

Comment:
I almost panicked at the `panic` but then I saw the `panic` above: I guess this is how we handle errors in this utility (which is fine)

Reply:
Yes...